deepchem/models/tensorgraph/models/graph_models.py

```diff
@@ -31,21 +31,21 @@ def __init__(self,
                mode="classification",
                **kwargs):
     """
-        Parameters
```

Comment:
The spacing is getting worse. Yapf hates our beautiful docstrings :-)

Reply:
maybe update yapf again?  Also one can fiddle with it I believe if you prefer certain things.